packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5476,25 +5473,33 @@ class WidgetToRenderBoxAdapter extends LeafRenderObjectWidget {
 ///
 /// See also:
 ///
-///  * [MouseTracker] an object that tracks mouse locations in the [GestureBinding].
-class Listener extends SingleChildRenderObjectWidget {
+///  * [Mouse], a similar widget that tracks mouse locations even when
+///    no button is pressed.
+class Listener extends StatelessWidget {
   /// Creates a widget that forwards point events to callbacks.
   ///
   /// The [behavior] argument defaults to [HitTestBehavior.deferToChild].
   const Listener({
     Key key,
     this.onPointerDown,
     this.onPointerMove,
-    this.onPointerEnter,
-    this.onPointerExit,
-    this.onPointerHover,
+    // We have to ignore the lint rule here in order to use deprecated
+    // parameters and keep backward compatibility. These parameters should be
+    // removed after v1.11 stable.
```

Comment:
this should be a TODO.

Why v1.11?

Reply:
I meant to use the next version for stable. I thought Flutter planned to bump minor every month, and release every quarter, so the next stable would be 1.11.

Reply:
I changed it to "After 2019Q3 stable". Is it better now?

Reply:
I would probably just say "after it goes to stable" it some such. who knows when we'll actually go to stable :-)